.devcontainer/devcontainer.json

```
@@ -0,0 +1,16 @@
+{
+	"name": "Foundry + Node",
```

```suggestion
	"name": "CoW AMM",
```

I suppose we could use different dev containers for different Foundry projects, and the name "Foundry" would be confusing.

---

In general, I would see the `.devcontainer/devcontainer.json` being part of a "smart contract template repository" from which we spawn. While there _may_ be a need to customise for the devcontainer for specific repositories, this may be a little premature customisation :smiley:

---

Sound reasonable. I'd still suggest a change to match other changes in this PR:

```suggestion
	"name": "Foundry",
```